src/main/java/firebase/sdk/dashboard/data/Platform.java

```diff
@@ -1,9 +1,25 @@
 package firebase.sdk.dashboard.data;
 
 public enum Platform {
-  ANDROID,
-  IOS,
-  WEB,
-  GAMES
+  UNKNOWN(0),
```

Comment:
I has used string for the constructor so we need to decide which one we will use so that we stay consistent.

Reply:
There is a toString method for enums already. If you call Platform.ANDROID.toString() it returns "ANDROID" and the same goes with the other platforms so I don't think the constructor needs to take in a string.